### PR TITLE
PMM-12643 consistent types in list output of pmm-admin

### PR DESCRIPTION
**Pull request by ademidoff**
PMM-12643

Link to the Feature Build: SUBMODULES-3445

### CODE REVIEW

File: admin/commands/list.go [hunk 1]
```diff
@@ -222,7 +222,7 @@ func (cmd *ListCommand) RunCmd() (Result, error) { //nolint:cyclop,maintidx
 		return "pull"
 	}
 	pmmAgentIDs := make(map[string]struct{})
-	var agentsList []listResultAgent
+	agentsList := make([]listResultAgent, 0, len(agentsRes.Payload.PMMAgent))
```
Comment (BupycHuk) [inline admin/commands/list.go:225]: there are much more items then just PMM Agents

Comment (ademidoff) [reply admin/commands/list.go]: Fixed.

**Comment by ademidoff** [comment]
> Can we add a test here?

Did you mean if we can add a test for RunCmd?

**Review by BupycHuk — APPROVED** [review]
wow, with these improvements code looks much more clear

**Review by JiriCtvrtka — APPROVED** [review]

**Pull request merged**